packages/host/tests/acceptance/code-mode-test.ts

```diff
@@ -1018,6 +1023,48 @@ module('Acceptance | code mode tests', function (hooks) {
         `[data-test-card-schema="Person"] [data-test-field-name="lastName"] [data-test-card-display-name="String"]`,
       )
       .exists();
+    assert
+      .dom(
+        `[data-test-card-schema="Person"] [data-test-field-name="title"] [data-test-card-display-name="String"]`,
+      )
+      .exists();
+    assert
+      .dom(
+        `[data-test-card-schema="Person"] [data-test-field-name="title"] .left .field-type`,
+      )
+      .hasText('Computed, Override');
+    assert
+      .dom(
+        `[data-test-card-schema="Person"] [data-test-field-name="title"] .right .computed-icon`,
+      )
+      .exists();
+
+    assert
+      .dom(
+        `[data-test-card-schema="Person"] [data-test-field-name="friend"] [data-test-card-display-name="Friend"]`,
+      )
+      .exists();
+    assert
+      .dom(
+        `[data-test-card-schema="Person"] [data-test-field-name="friend"] .left .field-type`,
+      )
+      .hasText('Linked');
+    assert
+      .dom(
+        `[data-test-card-schema="Person"] [data-test-field-name="friend"] .right .pill .linked-icon`,
+      )
+      .exists();
+
+    assert
+      .dom(
+        `[data-test-card-schema="Person"] [data-test-field-name="address"] [data-test-card-display-name="String"]`,
+      )
+      .exists();
+    assert
+      .dom(
+        `[data-test-card-schema="Person"] [data-test-field-name="address"] .left .field-type`,
```

Comment:
Using test selectors only would probably make this more robust

Reply:
Updated [236e9e5](https://github.com/cardstack/boxel/pull/718/commits/236e9e5a5ac90a6b21ad9c8eebcb88f59cdd5328)